index.js

```diff
@@ -94,6 +97,32 @@ Electron.prototype._spawn = function(url){
       case 'stderr': self.stderr.write(msg[1]); break;
     }
   });
+
+  // these event callbacks below will display the output and/or error 
+  // streams from an optional background process like Xvfb when running
+  // headless in a container, otherwise electron can fail silently
+  const errs = [], outs = [];
+  ps.stdout.on('data', function(data) {
+    outs.push(data)
+  });
+
+  ps.stderr.on('data', function(data) {
+    errs.push(data)
+  });
+
+  ps.on('close', function(code) {
+    const err = errs.join('').trim()
+    if (err.length) {
+      console.error(err)
+      return
+    }
+    var lst = outs
+      .join('')
+      .trim()
+      .split('\n')
+    var str = lst.slice(1).join('')
+    if (str) console.log(str)
```

Comment:
In the cases you were debugging, did you need stdout or stderr or both?

Reply:
Actually, just stderr, I can remove the stdout handling. I did see a font warning appended to the output message when using with xvfb. I think the warning was streamed with the stdout, but even that warning is not desirable in my use case.

Reply:
Notes:

1. Upon farther verification, the stdout is not needed, thanks for commenting on this.
2. It looks `electron-stream` itself, when piped to directly in the command line, does not need the `--no-sandbox` option to display an error message using xvfb in a debian-stretch Docker container.
3. Piping from `browser-run` to `electron-stream` does require the `ps.stderr.on(...)` callback in the same test container. So, not sure where the stream piping gets broken between xvfb, electron-stream, and browsr-run when `ps.stdout.on(...)` is not used in the electron-stream spawn function.

Testing details: 
$ echo "console.log('Hey from ' + location); window.close()" | npx [electron-stream|browser-run] [--no-sandbox]
[
  'this.sandbox=true',
  'opts.sandbox=undefined',
  'NO ps.stderr.on(), NO ps.stdout.on()' // same results as 'NO ps.stderr.on(), WITH ps.stdout.on()
]
**| electron-stream:** 
Hey from http://localhost:43359/

**| electron-stream --no-sandbox**
Hey from http://localhost:46449/

**| browser-run**
(!!! silent fail !!!)

**| browser-run --no-sandbox**
Hey from http://localhost:45019/

[
  'this.sandbox=true',
  'opts.sandbox=undefined',
  'WITH ps.stderr.on(), NO ps.stdout.on()' // same results as 'WITH ps.stderr.on(), WITH ps.stdout.on()' 
]
**| electron-stream:** 
Hey from http://localhost:43359/
Fontconfig warning: "/etc/fonts/fonts.conf", line 100: unknown element "blank"

**| electron-stream --no-sandbox**
Hey from http://localhost:46449/
Fontconfig warning: "/etc/fonts/fonts.conf", line 100: unknown element "blank"

**| browser-run**
[3176:0326/160110.360280:FATAL:atom_main_delegate.cc(194)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.

**| browser-run --no-sandbox**
Hey from http://localhost:45841/
Fontconfig warning: "/etc/fonts/fonts.conf", line 100: unknown element "blank"